crates/bioauth-flow-rpc/src/lib.rs

```diff
@@ -315,7 +338,7 @@ where
         let (opaque_liveness_data, signature) = self.sign(public_key.clone(), &liveness_data).await
             .map_err(EnrollError::Sign)?;
 
-        self.robonode_client
+        let response = self.robonode_client
```

Comment:
We might want to unpack the result here (and in similar places) right away

Reply:
https://github.com/humanode-network/humanode/pull/1026/commits/6ba5df193d4b034dcd274331f2b718bb85014438